lib/memory_profiler/reporter.rb

```diff
@@ -100,12 +106,19 @@ def object_list(generation)
           class_name = helper.lookup_class_name(klass)
           gem        = helper.guess_gem(file)
 
-          string     = '' << obj  if klass == String
+          if klass == String
+            if mem_lite
```

Comment:
This could be `if mem_lite && obj.size > 200` so that it only deals with large strings.

Reply:
Yeah, good call.  Was going to try something like this, but the #DayJob pulled me back in, so that took a bit of a back seat.